Epona/include/Epona/Analysis.hpp

```diff
@@ -293,6 +365,7 @@ double LineSegmentPointDistance(
 
```

Comment:
why LineSegmentPointDistance implementation in cpp file not here like others?

Reply:
Because it is not a template of a function. Therefore its definition cannot be in the include file. Otherwise, it would cause multiple declarations linking error.

Reply:
щл